src/components/SearchField/api/searchMovie.ts

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const API_KEY = process.env.REACT_APP_MOVIEDB_API_KEY;
+const API_SEARCH_MOVIE = "https://api.themoviedb.org/3/search/movie";
+
+const input = document.getElementById("searchInput") as HTMLInputElement | null;
+
```

Comment:
in my opinion, you do not need to use getelementbyid in react
https://ru.reactjs.org/docs/forms.html
probably here you also need to use the state
think about using useRef too